build.gradle

```diff
@@ -0,0 +1,106 @@
+buildscript {
```

Comment:
✏️ Might want to consider using the Kotlin DSL for gradle (ending up with `build.gradle.kts` files). The other small Codice projects using Gradle are using that DSL (granted: they also include Kotlin source code) and I suspect that in the future it will be the build file dialect we favor for Codice projects.